packages/ui-components/src/components/ActionsMenu.tsx

```diff
@@ -64,8 +63,14 @@ const StyledMenuIcon = styled(MoreVertIcon)`
   }
 `;
 
-export const ActionsMenu = ({ options }) => {
-  const [anchorEl, setAnchorEl] = React.useState(null);
+interface Option {
+  label: string;
+  action: () => void;
+  style?: CSSProperties;
+}
+
+export const ActionsMenu = ({ options }: { options: Option[] }) => {
```

Comment:
I had been adding `React.FC` but I just looked it up a bit more and I agree that you are right to not include it.

Reply:
FYI - I've added a section to the code style guide for typescript [here](https://beyond-essential.slab.com/posts/web-frontend-rewrite-03l47wk9#hkjg6-typescript)